server/src/main/java/org/elasticsearch/cluster/routing/allocation/RoutingAllocation.java

```diff
@@ -77,6 +83,13 @@ public class RoutingAllocation {
     @Nullable
     private final DesiredNodes desiredNodes;
 
+    private static final TimeValue CACHE_TTL = TimeValue.timeValueMinutes(1);
+    private static final Cache<RoutingNode, Long> unaccountableSearchableSnapshotSizes = CacheBuilder.<RoutingNode, Long>builder()
```

Comment:
I'd like to call the variable `unaccountedSearchableSnapshotSizes` instead. Also, please add javadoc describing what it means (searchable snapshots not yet registered in `ClusterInfo`).

Reply:
That indeed looks better! I've renamed it the field and the getter to `unaccountedSearchableSnapshotSizes` and added a comment to it.